$MirOS: ports/audio/libogg/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:42:57 tg Exp $
$OpenBSD: patch-configure,v 1.6 2003/12/14 00:52:22 naddy Exp $
--- configure.in.orig	Thu Sep 23 15:30:36 2004
+++ configure.in	Thu Nov  3 18:43:41 2005
@@ -38,7 +38,7 @@ if test -z "$GCC"; then
 		;;
         *)
                 DEBUG="-g"
-                CFLAGS="-O"
+                CFLAGS=""
                 PROFILE="-g -p" 
 		;;
         esac
@@ -61,8 +61,8 @@ else
 		;;
         *)
                 DEBUG="-g -Wall -fsigned-char"
-                CFLAGS="-O20 -fsigned-char"
-                PROFILE="-O20 -g -pg -fsigned-char" 
+                CFLAGS="-fsigned-char"
+                PROFILE="-g -pg -fsigned-char" 
 		;;
         esac
 fi
